packages/components/datepicker/src/index.ts

```diff
@@ -1,2 +1,4 @@
 export { Datepicker } from './Datepicker';
 export type { DatepickerProps } from './Datepicker';
+
+export * from './Calendar';
```

Comment:
I'm confused a bit about `Datepicker` and `Calendar` What's the difference?
Would you be able to provide an example of user's import use cases?

Reply:
Datepicker is a combobox, will be done in the follow up stories.